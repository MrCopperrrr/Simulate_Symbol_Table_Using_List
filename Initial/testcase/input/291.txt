BEGIN
BEGIN
BEGIN
INSERT a string
END
INSERT a number
LOOKUP a
END
INSERT a string
LOOKUP a
END
INSERT a number