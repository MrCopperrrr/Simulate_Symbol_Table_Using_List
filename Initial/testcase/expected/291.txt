success
success
2
success
1
success